### DISCUSSION TITLE
How to customize the rowsPerPage drop-down in Table pagination?

### DISCUSSION THREAD

**Discussion by fardisaz** [quasarframework/quasar#9492] [General - Components / Directives / etc] [answered]
I'd like to know how I can customize the rows number selection in table pagination.For example I only like to have 5,10 and 20 for drop-down. Is there any way which let me do that?

<img width="612" alt="Screenshot 2021-05-31 at 12 41 34" src="https://user-images.githubusercontent.com/33348251/120182809-2bcbb300-c20f-11eb-84aa-b87af051dbac.png">

**Comment by hawkeye64** [comment, marked as answer]
```js
export default {
  props: {
    pagination: Object,
    rowsPerPageOptions: {
      type: Array,
      default: () => [ 5, 7, 10, 15, 20, 25, 50, 0 ]
    }
  },

```
